backends/rapidpro/msg.go

```diff
@@ -504,6 +504,7 @@ type DBMsg struct {
 	SessionID_            SessionID  `json:"session_id,omitempty"`
 	SessionTimeout_       int        `json:"session_timeout,omitempty"`
 	SessionWaitStartedOn_ *time.Time `json:"session_wait_started_on,omitempty"`
+	SessionStatus_ string `json:"session_status,omitempty"`
```

Comment:
it seems whatever editor you're using doesn't run `gofmt`.. I really recommend vscode with the go extension if you haven't tried it

Reply:
Intellij. Fixed and pushed